CRM/Financial/BAO/FinancialType.php

```diff
@@ -396,10 +396,23 @@ public static function checkPermissionToEditFinancialType($financialTypeID) {
    *
    * @todo rename this function e.g isFinancialTypeACLsEnabled.
    *
+   * @deprecated since 5.75 will be removed around 5.90
```

Comment:
Should this have a deprecatedWarning call as well?

Reply:
We actually have core calls to it still - so the removal date is probably optimistic but we can always push it back